src/test/java/com/blackduck/integration/sca/upload/client/UploaderConfigTest.java

```diff
@@ -10,14 +10,15 @@
 import java.util.Properties;
 
 import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Assertions;
```

Comment:
we already import the assert methods explicitly above (lines 3-4). Do we need this change or was this brought in via the optimizations?

Reply:
It came in via the package path change, but I don't know why. I am updating.